src/components/Comments/Comment.js

```diff
@@ -327,14 +328,14 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(12),
-    color: COLORS.blackgrey,
+    color: '#828282',
     lineHeight: 14,
     marginLeft: 16
   },
   post: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(14),
-    color: COLORS.mineShaft,
+    color: '#333333',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text colors have been updated. Ensure these changes are consistent with the overall design and color scheme of the application.
```diff
- color: COLORS.blackgrey,
+ color: '#828282',
- color: COLORS.mineShaft,
+ color: '#333333',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->